src/Frontend/Modules/Blog/Actions/Detail.php

```diff
@@ -115,10 +114,13 @@ private function buildForm(): void
         $this->form = new FrontendForm('commentsForm');
         $this->form->setAction($this->form->getAction() . '#' . FL::act('Comment'));
 
+        $cookie = FrontendModel::getContainer()->get('fork.cookie');
         // init vars
-        $author = (CommonCookie::exists('comment_author')) ? CommonCookie::get('comment_author') : null;
-        $email = (CommonCookie::exists('comment_email') && \SpoonFilter::isEmail(CommonCookie::get('comment_email'))) ? CommonCookie::get('comment_email') : null;
-        $website = (CommonCookie::exists('comment_website') && \SpoonFilter::isURL(CommonCookie::get('comment_website'))) ? CommonCookie::get('comment_website') : 'http://';
+        $author = $cookie->get('comment_author');
+        $email = ($cookie->has('comment_email') && \SpoonFilter::isEmail($cookie->get('comment_email')))
```

Comment:
Wouldn't it be better to use `filter_var($cookie->get('comment_email'), FILTER_VALIDATE_EMAIL)`?